### ISSUE TITLE
windows + msys2 : compilation error

### ISSUE THREAD

**Issue by vtorri** [thorvg/thorvg#1830]
```
[35/85] Compiling C++ object src/libthorvg-0.dll.p/loaders_lottie_tvgLottieBuilder.cpp.obj
FAILED: src/libthorvg-0.dll.p/loaders_lottie_tvgLottieBuilder.cpp.obj
"c++" "-Isrc/libthorvg-0.dll.p" "-Isrc" "-I../src" "-Iinc" "-I../inc" "-I." "-I.." "-Isrc/bindings/capi" "-I../src/bindings/capi" "-Isrc/renderer" "-I../src/ren
derer" "-Isrc/renderer/sw_engine" "-I../src/renderer/sw_engine" "-Isrc/common" "-I../src/common" "-Isrc/loaders" "-I../src/loaders" "-Isrc/loaders/tvg" "-I../sr
c/loaders/tvg" "-Isrc/loaders/svg" "-I../src/loaders/svg" "-Isrc/loaders/lottie" "-I../src/loaders/lottie" "-Isrc/loaders/external_png" "-I../src/loaders/extern
al_png" "-Isrc/loaders/external_jpg" "-I../src/loaders/external_jpg" "-Isrc/loaders/raw" "-I../src/loaders/raw" "-Isrc/savers" "-I../src/savers" "-Isrc/savers/t
vg" "-I../src/savers/tvg" "-Isrc/savers/gif" "-I../src/savers/gif" "-IC:/Documents/msys2/mingw64/include/libpng16" "-IC:/Documents/msys2/mingw64/include" "-fvis
ibility=hidden" "-fdiagnostics-color=always" "-D_FILE_OFFSET_BITS=64" "-Wall" "-Winvalid-pch" "-std=c++14" "-Os" "-g" "-DEXAMPLE_DIR=\"C:/Documents/msys2/home/v
incent.torri/gitroot_64/thorvg_gh/src/examples/resources\"" "-DTEST_DIR=\"C:/Documents/msys2/home/vincent.torri/gitroot_64/thorvg_gh/test/resources\"" "-DTVG_EX
PORT" "-DTVG_BUILD" "-fno-exceptions" "-fno-rtti" "-fno-stack-protector" "-fno-math-errno" "-fno-unwind-tables" "-fno-asynchronous-unwind-tables" "-Woverloaded-
virtual" "-Wno-unused-parameter" -MD -MQ src/libthorvg-0.dll.p/loaders_lottie_tvgLottieBuilder.cpp.obj -MF "src/libthorvg-0.dll.p/loaders_lottie_tvgLottieBuilde
r.cpp.obj.d" -o src/libthorvg-0.dll.p/loaders_lottie_tvgLottieBuilder.cpp.obj "-c" ../src/loaders/lottie/tvgLottieBuilder.cpp
../src/loaders/lottie/tvgLottieBuilder.cpp: In function 'void _buildReference(LottieComposition*, LottieLayer*)':
../src/loaders/lottie/tvgLottieBuilder.cpp:957:13: error: 'strcmp' was not declared in this scope
  957 |         if (strcmp(layer->refId, (*asset)->name)) continue;
      |             ^~~~~~
../src/loaders/lottie/tvgLottieBuilder.cpp:30:1: note: 'strcmp' is defined in header '<cstring>'; did you forget to '#include <cstring>'?
   29 | #include "tvgTaskScheduler.h"
  +++ |+#include <cstring>
   30 |


```

**Comment by hermet** [comment]
@vtorri Hello, +#include <cstring> this would fix it?

**Comment by vtorri** [comment]
../src/savers/gif/tvgGifSaver.cpp:138:18: error: 'strdup' was not declared in this scope

**Comment by vtorri** [comment]
#include <cstring>

for both files

**Comment by vtorri** [comment]
i'm at work, ssh is not allowed

**Comment by vtorri** [comment]
otherwise i would have done a PR of course

**Comment by vtorri** [comment]
fixed in #1831
